### ISSUE TITLE
Aplikacia: vypocet vysky 2-3% dane 

### ISSUE THREAD

**Issue by HanaBene** [slovensko-digital/navody.digital#202]
pridat kalkulacku do navodov na darovanie casti dane, kde zadas sumu zaplatenu a podla toho ci mozes darovat 2 alebo 3 percenta ti vypocita konkretnu sumu.

**Comment by smith558** [comment]
@HanaBene Nieco taketo? [http://gramotnafinancnost.sk/2percentazdani/](url)

**Comment by HanaBene** [comment]
ano, to vyzera dobre, ked to bude vediet byt okno zasadene do navodu, bude
parada.
Hana Benediktová | Návody.Digital
hana.benediktova@slovensko.digital



so 8. 6. 2019 o 16:06 Stanley Modrak <notifications@github.com> napísal(a):

**Comment by HanaBene** [comment]
toto mame ako feature v aplikacii danove priznanie, mohli by sme to modifikovat a vytiahnut ako samostatnu appku pre zamestnancov, ktori si podavaju 2% sami dodatocne

**Comment by TomasMakan** [comment]
Spravil som si krátky prieskum toho, čo je dostupné pre 2% na internete. Všade sú len textové návody ( niektoré veľmi dobre popísané), ale stále sa bavíme o niekoľko stranách textu a možnostiach, ktoré si musí človek prejsť, aby mal aspoň základnú predstavu, ako postupovať. 
Myslím, že toto je ideálny adept na ďalšiu appku v štýle priznania. Z toho, čo som si načítal by to malo byť o dosť jednoduchšie ako priznanie a zároveň sa na to dá vyskladať flow (napr.: Som FO ->zadaj sumu z vyúčtovania ->mám dobrovoľníctvo -> chcem poukázať tomuto OZ = vyplnené PDF)
